src/AudioUnit/AudioComponent.cs

```diff
@@ -224,6 +379,58 @@ public XamCore.AppKit.NSImage GetIcon ()
 			return new XamCore.AppKit.NSImage (AudioComponentGetIcon (handle));
 		}
 #endif
+		[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+		[DllImport (Constants.AudioUnitLibrary, EntryPoint = "AudioUnitExtensionSetComponentList")]
```

Comment:
it's not public so there's no need to have an `EntryPoint`, so name the method with it's native name